django/db/models/functions/datetime.py

```diff
@@ -243,9 +243,8 @@ class TruncDate(TruncBase):
     kind = 'date'
     lookup_name = 'date'
 
-    @cached_property
-    def output_field(self):
-        return DateField()
+    def __init__(self, *args, output_field=None, **kwargs):
```

Comment:
It's the current behavior https://github.com/django/django/blob/5debbdfcc84266703191e084914998e38f5f52eb/tests/db_functions/test_datetime.py#L685.